dev/ToastNotifications/ToastNotificationManager.cpp

```diff
@@ -0,0 +1,115 @@
+﻿#include "pch.h"
+#include "ToastNotificationManager.h"
+#include "Microsoft.Windows.ToastNotifications.ToastNotificationManager.g.cpp"
+#include <winrt/Windows.ApplicationModel.Core.h>
+#include <winrt/Windows.Storage.h>
+#include <winrt/base.h>
+#include "ToastActivationCallback.h"
+#include "externs.h"
+#include "ToastNotificationUtility.h"
+#include <frameworkudk/pushnotifications.h>
+
+static winrt::event<winrt::Windows::Foundation::EventHandler<winrt::Microsoft::Windows::ToastNotifications::ToastActivatedEventArgs>> g_toastHandlers;
+
+static int g_toastHandlerCount = 0;
```

Comment:
Curious if we really need this variable? Can't we infer this from g_toastHandlers assuming it has a Size() method?

Reply:
There is no size method but there is a bool to check registered delegates which would cleaner.

https://docs.microsoft.com/en-us/uwp/cpp-ref-for-winrt/event#eventoperator-bool